### PR TITLE
Download all plugins from repository

### PR DESCRIPTION
**Pull request by eigengravy**

### CODE REVIEW

File: app/src/main/java/com/lagradost/cloudstream3/utils/AppUtils.kt [hunk 1]
```diff
@@ -257,6 +259,26 @@ object AppUtils {
...
+            downloadAllPluginsDialog(url)
+        }
+    }
+
+    fun Activity.downloadAllPluginsDialog(repositoryUrl: String) {
+        runOnUiThread {
+            val context = this
+            val builder: AlertDialog.Builder = AlertDialog.Builder(this)
+            builder.setTitle(
+                "Download all plugins from this repo?"
```
Comment (Blatzar) [inline app/src/main/java/com/lagradost/cloudstream3/utils/AppUtils.kt:271]: R.string :skull:

Comment (Blatzar) [reply app/src/main/java/com/lagradost/cloudstream3/utils/AppUtils.kt:271]: Probably good to include the repo name if that's available without another request :thinking:

Comment (eigengravy) [reply app/src/main/java/com/lagradost/cloudstream3/utils/AppUtils.kt]: Done 🚀

**Comment by eigengravy** [comment]
i certainly did not

**Comment by LagradOst** [comment]
You can use Download and Cancel instead of yes and no. that is more readable for the user impo

**Comment by eigengravy** [comment]
Updated it

**Pull request merged**